package.json

```diff
@@ -28,7 +28,7 @@
     "format": "prettier --write .",
     "lint": "lerna run lint --scope @mux-elements/*",
     "pretest": "yarn build:packages",
-    "test": "lerna run test --scope @mux-elements/*",
+    "test": "lerna run test --scope @mux-elements/mux-player",
```

Comment:
this change is purely demonstrative and must be backed out before this can proceed.